app/assets/javascripts/extractors.js

```diff
@@ -105,6 +114,34 @@ $(document).ready(function () {
         });
     });
 
+    // Try regular expression against example.
+    $(".xtrc-try-regex-replace").on("click", function () {
+        var button = $(this);
+
+        button.html("<i class='fa fa-spinner fa-spin'></i> Trying...");
+        var promise = testRegexReplace($("#regex").val(), $("#replacement").val(), $("#replace_all").is(":checked"));
+
+        promise.done(function (matchResult) {
+            if (matchResult.finds) {
+                if (matchResult.match != null) {
+                    highlightMatchResult(matchResult);
```

Comment:
I've changed this to show the result in a separate `div`. I'd come back to your styling skills, though. :wink: